docs/docs/run-node/02-interact-node.md

```diff
@@ -88,8 +88,8 @@ It's also possible to execute an RPC call to query the node for information:
 
 ```bash
 grpcurl \
-    -plaintext
-    -d '{"address":"$MY_VALIDATOR"}' \
+    -plaintext \
+    -d "{\"address\":\"$MY_VALIDATOR_ADDRESS\"}" \
```

Comment:
- to be consistent with the env naming which was `MY_VALIDATOR_ADDRESS`
- The env var was not passed to the cmd properly